### DISCUSSION TITLE
Restore lock screen backgrounds

### DISCUSSION THREAD

**Discussion by Bilge** [ChrisTitusTech/winutil#3026] [Q&A] [answered]
I disabled telemetry and lost the nice rotating background images on the lock screen. How can I get just those back?

**Comment by zmileski** [comment]
You can just intall "Dynamic Theme" app it is better alternative
but if you want windows spotlight
You will need to enable backround apps
and re-register windows spotlight

1. **Reset Spotlight: Navigate to** 

C:\Users\YourUsername\AppData\Local\Packages\Microsoft.Windows.ContentDeliveryManager_cw5n1h2txyewy\LocalState\Assets
C:\Users\YourUsername\AppData\Local\Packages\Microsoft.Windows.ContentDeliveryManager_cw5n1h2txyewy\Settings
Delete the  settings.dat and  roaming.lock  files.

or command:
DEL /F /S /Q /A "%USERPROFILE%/AppData\Local\Packages\Microsoft.Windows.ContentDeliveryManager_cw5n1h2txyewy\LocalState\Assets"
DEL /F /S /Q /A "%USERPROFILE%/AppData\Local\Packages\Microsoft.Windows.ContentDeliveryManager_cw5n1h2txyewy\Settings"

2. **Re-register Spotlight: Open PowerShell as administrator and enter the following command:** 

PowerShell -ExecutionPolicy Unrestricted -Command "& {$manifest = (Get-AppxPackage *ContentDeliveryManager*).InstallLocation + '\AppxManifest.xml' ; Add-AppxPackage -DisableDevelopmentMode -Register $manifest}"

3. Check your settings: Go to Settings > Personalization > Lock Screen. Ensure that Windows Spotlight is selected under the Background dropdown menu. If it is, try switching to Picture or Slideshow, wait for a moment, and then switch back to Windows Spotlight.

**Comment by Bilge** [comment, marked as answer]
I really appreciate your comprehensive response, but I actually was able to just go to Personalise > Lock Screen and switch it back to _Windows spotlight_.
